### DISCUSSION TITLE
How many epochs did you use to train?

### DISCUSSION THREAD

**Discussion by hyeseojy** [InterDigitalInc/CompressAI#111] [Q&A] [answered]
Thank you for your amazing work!

How many epochs did you use to train?
You already wrote down the steps you used on the website, but as you know, the code works based on epoch.
When I calculated based on the Vimeo dataset, it was about 400 epochs. Is it correct?

Thanks again for your excellent work. Looking forward to your reply.
Thank you!

**Comment by fracape** [comment, marked as answer]
Hi, 
Yes it's correct, it's in this ballpark, although the number of epochs usually depends on a termination criterion, as we mention in the [paper](https://arxiv.org/pdf/2011.03029.pdf):
"Training Unless specified otherwise, the provided pre-trained networks were trained for 4-5M steps on
256 × 256 image patches randomly extracted and cropped from the Vimeo-90K dataset [22].
Models were trained with a batch size of 16 or 32, and an initial learning rate of 1e-4 for approximately
1-2M steps. The learning rate is then divided by 2 whenever the evaluation loss reaches a plateau (we use a
patience of 20 epochs)."
